system/doc/reference_manual/errors.xml

```diff
@@ -109,26 +109,29 @@
       and a stack trace (which aids in finding the code location of
       the exception).</p>
     <p>The stack trace can be bound to a variable from within
-      a <c>try</c> expression, and is returned for
-      exceptions of class <c>error</c> from a <c>catch</c> expression.</p>
+      a <c>try</c> expression.</p>
     <p>An exception of class <c>error</c> is also known as a run-time 
       error.</p>
 
       <section>
 	<marker id="stacktrace"></marker>
 	<title>The call-stack back trace (stacktrace)</title>
         <p>The stack back-trace (<em>stacktrace</em>) is a list of
-          <c>{Module,Function,Arity,Location}</c>
+          <c>{Module,Function,Arity,ExtraInfo}</c>
           tuples. The field <c>Arity</c> in the first tuple can be the
           argument list of that function call instead of an arity integer,
           depending on the exception.</p>
-
-	<p><c>Location</c> is a (possibly empty) list of two-tuples
-	that can indicate the location in the source code of the
-	function.  The first element is an atom describing the type of
-	information in the second element. The following items can
-	occur:</p>
+	  <p><c>ExtraInfo</c> is a (possibly empty) list of two-tuples
```

Comment:
Does this two- refer to two elements?
If so would "two-element tuples" convey a better message?